gpx.cc

```diff
@@ -494,7 +494,7 @@ tag_cache_desc(const QXmlStreamAttributes& attr)
 {
   cache_descr_is_html = 0;
   if (attr.hasAttribute("html")) {
-    if (attr.value("html").toString().compare("True") == 0) {
+    if (attr.value("html").toString().compare(QString("True")) == 0) {
```

Comment:
QStringLiteral() here (and throughout) would save some alloc/free cycles.

Also, please report which qt5 is "latest" in your commit message.